src/System.IO.Compression.Brotli/src/Interop/Interop.Brotli.Decoder.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Any reason this file isn't in src\Common\Interop, that LibNameDecoder isn't in Interop.Library, etc.?

Reply:
Since they're interop-ing into our own binaries it felt weird to put them into src\Common\Interop. [We have the zlib interop declarations under System.IO.Compression and not in Common](https://github.com/dotnet/corefx/blob/master/src/System.IO.Compression/src/Interop/Interop.zlib.Unix.cs) so I was also following that pattern, though I don't have a great reason for not putting them into common instead.

Reply:
I think it is fine to not have these under src\Common.

It would be nice to cleanup the few bits from these bits that are under Common for consistency:

>  src\Common\src\Interop\Unix\Interop.Libraries.cs(15):        internal const string CompressionNative = "System.IO.Compression.Native";
>  src\Common\src\Interop\Windows\Interop.Libraries.cs(33):        internal const string Zlib = "clrcompression.dll";

Reply:
>It would be nice to cleanup the few bits from these bits that are under Common for consistency:

Yup! I'm changing both of these to CompressionNative so I can get rid of the local "library.{os}.cs" files that I have.